test/migrators/calc_interpolation/calc_remove_interpolation.hrx

```diff
@@ -18,6 +18,9 @@ $d: 5;
 // Nested and more interpolations
 .a { .b: calc(#{$b} + max(#{$c, 2})); }
 
+// var() nested
+.a { .b: calc(var(#{$b}) + #{$c + 2} + var(#{$d})); }
+
```

Comment:
Add a test for something like  `var(--prefix-#{$variable}-suffix)` mentioned above.

Reply:
👍